components/eamxx/src/share/atm_process/atmosphere_process.hpp

```diff
@@ -608,6 +604,12 @@ class AtmosphereProcess : public ekat::enable_shared_from_this<AtmosphereProcess
 
   // IOP object
   iop_data_ptr m_iop_data_manager;
+
+  // The list of in/out field/group requests.
+  std::list<FieldRequest>   m_required_field_requests;
+  std::list<FieldRequest>   m_computed_field_requests;
+  std::list<GroupRequest>   m_required_group_requests;
+  std::list<GroupRequest>   m_computed_group_requests;
```

Comment:
These need to be lists if we want to easily edit them. I don't think it's a problem if we process duplicates.